app/assets/stylesheets/modules/_notifications.styl

```diff
@@ -32,7 +32,7 @@
   position relative
   z-index 9999
   width 100%
-  text-align left
+  text-align center
```

Comment:
You can't be confident that this change won't mess up other places in the UI that use this class.

I suggest, instead, that you either add an existing class to the html that will take precedence, or add a new class and use it.